custom-login/src/App.jsx

```diff
@@ -37,20 +37,25 @@ const App = () => {
     history.push('/login');
   };
 
+  const onAuthResume = async () => {
+    history.push('/login');
+  };
+
   const [corsErrorModalOpen, setCorsErrorModalOpen] = React.useState(false);
 
   return (
     <Security
       oktaAuth={oktaAuth}
       onAuthRequired={customAuthHandler}
+      onAuthResume={onAuthResume}
```

Comment:
From code of `okta-react` looks like `onAuthResume` has no effect on `Security` component, only does on `LoginCallback `

Reply:
Updated